CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+We welcome your contributions!
+
+_Each PR needs at least one approving review before merge._
```

Comment:
Just a thought: Is one approval enough?

Reply:
at least one :) 

I prefer that couple devs look at my code